api/app/services/RoutingService.scala

```diff
@@ -7,34 +7,44 @@ import com.trifectalabs.roadquality.v0.models.{ Point, MapRoute }
 import com.trifectalabs.polyline.{ Polyline, LatLng }
 import db.dao.MapDao
 import scala.concurrent.{ExecutionContext, Future}
+import play.api.libs.ws._
+import play.api.libs.json._
+import play.api.Configuration
 
 trait RoutingService {
-  def generateRoute(startLat: Double, startLng: Double, endLat: Double, endLng: Double): Future[MapRoute]
-  def generateRoute(points: Seq[Point]): Future[MapRoute]
+  def generateRoute(points: String): Future[MapRoute]
   def snapPoint(point: Point): Future[Point]
 }
 
-class RoutingServiceImpl @Inject()(mapDao: MapDao)(implicit ec: ExecutionContext) extends RoutingService {
-
-  def generateRoute(startLat: Double, startLng: Double, endLat: Double, endLng: Double): Future[MapRoute] = {
-    mapDao.route(
-      Point(startLat, startLng),
-      Point(endLat, endLng)
-    )
-  }
-
-  def generateRoute(points: Seq[Point]): Future[MapRoute] = {
-    Future.sequence {
-      (points.init zip points.tail).map { case (p1, p2) =>
-        mapDao.route(p1, p2)
-      }
-    } map { b: Seq[MapRoute] =>
-      val pl = Polyline.encode(b.flatMap(p => Polyline.decode(p.polyline)).toList)
-    MapRoute(polyline = pl, distance = b.map(_.distance).sum ) }
+class RoutingServiceImpl @Inject()(configuration: Configuration, ws: WSClient)(implicit ec: ExecutionContext) extends RoutingService {
+  lazy val osrmRoutingUri = configuration.getString("osrm.routing.uri").get
+  lazy val osrmNearestUri = configuration.getString("osrm.nearest.uri").get
+
+  def generateRoute(points: String): Future[MapRoute] = {
+    val osrmUrl = s"$osrmRoutingUri/$points"
+    ws.url(osrmUrl).get().map { res =>
```

Comment:
what happens if this fails? we need to handle the case that OSRM is down - i.e., return a 503 and handle gracefully on the client